src/Shouldly/ShouldStaticClasses/ShouldThrowExtensionsOverloads.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using JetBrains.Annotations;
+
+namespace Shouldly
+{
+    public static partial class Should
```

Comment:
Good point, is this better:

    public static class ShouldThrowOverloads
    {
        public static TException ShouldThrow<TException>(this Action actual) where TException : Exception
        {
            return Should.Throw<TException>(actual);
        }
    ...